src/index.html

```diff
@@ -11,6 +11,20 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="big-search-bar" data-qa="big">
+      <input
+        class="big-search-bar__input"
+        type="search"
+        placeholder="Try “Los Angeles”"
+        data-qa="keypress"
+      >
+    </form>
+    <form class="small-search-bar" data-qa="small">
+      <input
+        class="small-search-bar__input"
+        type="search"
+        placeholder="Try “Los Angeles”"
```

Comment:
```suggestion
        placeholder="Try &quot;Los Angeles&quot;"
```
Use ` &quot;` instead